dev/list_changed_files.py

```diff
@@ -9,7 +9,8 @@
 """
 import argparse
 import os
-import requests
+import urllib.request
+import json
```

Comment:
`requests` is not a built-in packages. By replacing it, we can run this script quicker without package installation.